old_configs/v0.18.4.yaml

```diff
@@ -0,0 +1,220 @@
+# startup_input is how you configure the UI.  It's literally just a set of keys
```

Comment:
cool, but remove the comments please (to match the other files in this directory).

Reply:
Thanks, comments have been fixed